fe/fe-core/src/main/java/com/starrocks/common/Config.java

```diff
@@ -687,6 +687,19 @@ public class Config extends ConfigBase {
      */
     @ConfField(mutable = true)
     public static int alter_table_timeout_second = 86400; // 1day
+
+    /**
+     * The alter handler max worker threads
+     */
+    @ConfField
+    public static int alter_max_worker_threads = 4;
+
+    /**
+     * The alter handler max queue size for worker threads
+     */
+    @ConfField
+    public static int alter_max_worker_queue_size = 4096;
```

Comment:
1. there are no queue before, `be * alter_tablet_worker_count` concurrent request will acquire database write lock
2. it will not clear task, all queued request will be handled and failed since new tablet will be droped